src/ucp/wireup/select.c

```diff
@@ -1096,24 +1130,20 @@ ucp_wireup_add_am_bw_lanes(const ucp_wireup_select_params_t *select_params,
     bw_info.usage             = UCP_WIREUP_LANE_USAGE_AM_BW;
 
     /* am_bw_lane[0] is am_lane, so don't re-select it here */
+    am_lane = UCP_NULL_LANE;
     for (lane_desc_idx = 0; lane_desc_idx < select_ctx->num_lanes; ++lane_desc_idx) {
         if (select_ctx->lane_descs[lane_desc_idx].usage & UCP_WIREUP_LANE_USAGE_AM) {
-            addr_index                 = select_ctx->lane_descs[lane_desc_idx].addr_index;
-            rsc_index                  = select_ctx->lane_descs[lane_desc_idx].rsc_index;
-            bw_info.md_map            |= UCS_BIT(context->tl_rscs[rsc_index].md_index);
-            bw_info.local_dev_bitmap  &= ~UCS_BIT(context->tl_rscs[rsc_index].dev_index);
-            bw_info.remote_dev_bitmap &= ~UCS_BIT(select_params->address->
-                                                  address_list[addr_index].dev_index);
-            break; /* do not continue searching due to we found
-                      AM lane (and there is only one lane) */
+            /* do not continue searching since we found AM lane (and there is
+             * only one am lane) */
```

Comment:
[minor (feel free to ignore and fix if any other comemnts)] inconsistency for AM lane:
1136 line - "... AM lane ..."
1137 line - "... am lane ..."